theme/theme.inc

```diff
@@ -60,6 +60,11 @@ function islandora_preprocess_islandora_default_edit(array &$variables) {
           'datastream' => $ds,
         )),
       );
+    }
+    else if (user_access(ISLANDORA_VIEW_DATASTREAM_HISTORY)) {
```

Comment:
Can't just be an else, thats why the whole thing was wrapped in an if, this statements needs to happen is the user has access to ISLANDORA_VIEW_DATASTREAM_HISTORY, but not access to the datastream.